ext/js/pages/settings/dictionary-controller.js

```diff
@@ -775,9 +893,27 @@ export class DictionaryController {
         }
     }
 
+    /** */
+    async _checkForUpdates() {
```

Comment:
If someone checks for updates, closes the settings page and comes back, I assume the "Update available" state will disappear?

Reply:
Yeah, if you reload you'll need to press the button to check for updates again, but it's pretty fast.